perun-openapi/openapi.yml

```diff
@@ -4743,9 +4759,47 @@ paths:
   #                                               #
   #################################################
 
-  #TODO createExtSource
+  /json/extSourcesManager/createExtSource/es:
+    post:
+      tags:
+        - ExtSourcesManager
+      operationId: createExtSourceFromExtSourceObject
+      summary: "Creates an external source. ExtSource object must contain: name, type. Other parameters are ignored."
+      parameters:
```

Comment:
When you are calling method with the `/json/` serializer, you have to pass the data in the request body, not in url parameters.